app/objects/secondclass/c_fact.py

```diff
@@ -20,15 +24,39 @@
 }
 
 
+class Restriction(Enum):
+    UNIQUE = 0
+    SINGLE = 1
```

Comment:
@ArtificialErmine if this is more about how many times a fact should be used, and not what we've called "restrictions" in our internal notes, would a better name for this be `Limit`?

Reply:
I think that makes sense, though we'd need to change the values of the enum objects accordingly (or likely just turn the whole thing into a 'limit_count` field). Restrictions as defined in the notes seem like they'll be more apart of the service than the object at the moment, at least from my perspective.

Reply:
Renamed the field to `limit_count` with a default value of `-1` as an indication of "this Fact can be used an unlimited amount of times"